tools/ci_build/github/azure-pipelines/nuget/cpu-esrp-featruizers-pipeline.yml

```diff
@@ -6,6 +6,9 @@ schedules:
     - master
   always: true
 
+variables:
+  PackageName: 'Microsoft.ML.ORTFeaturizers'
```

Comment:
Should it be a parameter instead of variable?

Reply:
The Solution which we use to produce the package requires an env var to run the tests. Currently, it is the only way to override the default package name.

Reply:
Then why do you need the parameter?

Reply:
But please be very careful, it may not work act you think:

"Variables are also injected into the environment. This includes both variables defined at the step level, and variables defined at the job or stage level, with the most locally scoped variable winning. When variables are turned into environment variables, variable names become uppercase, and periods turn into underscores. For example, the variable any.variable becomes $ANY_VARIABLE."

Reply:
yes, we've been bit by the uppercasing of env variables before. and we learnt it the hard way. Jignesh and I spent quite sometime debugging this upper casing stuff last year when it wasn't documented.

Reply:
I feel fine if this file is only temporary.

Reply:
That is true. I think it is completely messed up. But to change it we'd need to change the project and GPU, MKLML and possibly other pipelines.